packages/flutter/lib/src/painting/binding.dart

```diff
@@ -53,7 +53,20 @@ mixin PaintingBinding on BindingBase, ServicesBinding {
   /// See also:
   ///
   ///  * [ShaderWarmUp], the interface of how this warm up works.
-  static ShaderWarmUp shaderWarmUp = const DefaultShaderWarmUp();
+  static ShaderWarmUp get shaderWarmUp => _shaderWarmUp;
+  static void set shaderWarmUp(ShaderWarmUp value) {
```

Comment:
nit: This is failing the analyzer. "Avoid return types on setters"

Reply:
Thank you very much for the suggestion! Done.